File: python/questao2.py
def converter_temperatura(kelvin):
    celsius = kelvin - 273.15
    fahrenheit = (kelvin - 273.15) * 9/5 + 32
    print(f"Temperatura em Celsius: {celsius:.2f}")
    print(f"Temperatura em Fahrenheit: {fahrenheit:.2f}")

temperatura_kelvin = float(input("Digite a temperatura em Kelvin: "))
converter_temperatura(temperatura_kelvin)

File: c/questao2.c
#include <stdio.h>

void converterTemperatura(float kelvin) {
    float celsius = kelvin - 273.15;
    float fahrenheit = (kelvin - 273.15) * 9/5 + 32;
    printf("Temperatura em Celsius: %.2f\n", celsius);
    printf("Temperatura em Fahrenheit: %.2f\n", fahrenheit);
}

int main() {
    float temperaturaKelvin;

    printf("Digite a temperatura em Kelvin: ");
    scanf("%f", &temperaturaKelvin);

    converterTemperatura(temperaturaKelvin);
    return 0;
}
